Makefile

```diff
@@ -79,6 +98,7 @@ help-variables:
 	@echo
 	@echo  '- PREFIX       (directory)  : Installation prefix (default: /usr/local)'
 	@echo  '- KEEP_SYMBOLS (1 or 0)     : Whether to keep symbols (default: 0)'
+	@echo  '- DEBUG        (1 or 0)     : Whether to build with debug information (default: 0)'
```

Comment:
Probably this should be documented in https://lima-vm.io/docs/dev/ ? Maybe create the `/docs/dev/debug` page.

Reply:
If I were to write a document on debugging in Visual Studio Code, I’d like to improve how `ha.pid` is handled to reduce the need for `preLaunchTask` and `postDebugTask`.